crates/database/src/orders.rs

```diff
@@ -174,8 +174,8 @@ pub async fn read_order(
     id: &OrderUid,
 ) -> Result<Option<Order>, sqlx::Error> {
     const QUERY: &str = r#"
-SELECT * FROM ORDERS
-WHERE uid = $1
+        SELECT * FROM ORDERS
+        WHERE uid = $1
```

Comment:
I had this indented to the left on purpose so larger sql statements become more readable. But doesn't really matter.